README.md

```diff
@@ -1,8 +1,38 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+[![Build Status](https://github.com/blyscuit/docs/actions/workflows/deploy_fly.yml/badge.svg)](https://github.com/blyscuit/rails-ic/tree/chore/3-deployment-automation)
 
 ## Introduction
 
-> *App introduction goes here ...*
+Rails Internal Certificate.
+
+
+## Get Started
+
+### Requirements
+
+- Install ruby and set your local ruby version to `3.0.1`
+- Install rails `7.0.1`
+- Install node `16.13.2` (For creating web application)
+
+> 📝 If running on Apple M1, to build docker image, please make sure to set platform to AMD64 by `export DOCKER_DEFAULT_PLATFORM=linux/amd64`
+
+### Use Script
+
+```sh
+make dev
+```
+
+### Manually
+
+```
+bundle install
+bin/docker-prepare
+bin/rails s
+```
+
+## Deployment
+
+Staging URL: https://rails-ic-staging.fly.dev/
+Production URL: https://rails-ic.fly.dev/
```

Comment:
Should we move these at the top of the readme file (maybe in the `introduction` part)? 

usually, public web app will have the link to that app at the very top — as many `potential users` might just look for that link 😇 

Example with Amplication:
![image](https://github.com/blyscuit/rails-ic/assets/77609814/d98ff446-574e-4457-91e1-5e1b28f1632e)